doc/simplestreams-metadata.txt

```diff
@@ -141,15 +141,15 @@ For a shared directory, use a URL of the form "file:///sharedpath".
 
 3. Provider specific storage
 
-Providers may allow additional locations to search for metadata and tools. For Openstack, keystone
+Providers may allow additional locations to search for metadata and agent binaries. For Openstack, keystone
 endpoints may be created by the cloud administrator. These are defined as follows:
 
 juju-tools      : the <path_url> value as described above in Tools Metadata Contents
 product-streams : the <path_url> value as described above in Image Metadata Contents
 
 4. Central web location (https://streams.canonical.com/juju)
 
-This is the default location used to search for image and tools metadata and is used if no matches
+This is the default location used to search for image and agent binaries metadata and is used if no matches
```

Comment:
used to search for image and agent binary metadata
(I think its being used as an adjective here)